src/doc/reference/glossary/serial_execution.md

```diff
@@ -0,0 +1,5 @@
+---
+title: Serial execution
+tags: serial execution
+---
+Execution of the {% defn "serialization" %} of a Cilk program.
```

Comment:
Since the "serialization" entry just points to "serial projection," should this reference be "serial projection," instead of "serialization"?